library/Fission/User/CID.hs

```diff
@@ -1,7 +1,9 @@
 module Fission.User.CID
-  ( module Fission.User.CID.Query
-  , module Fission.User.CID.Mutation
+  ( module Fission.User.CID.Retriever
+  , module Fission.User.CID.Creator
+  , module Fission.User.CID.Destroyer
   ) where
 
-import Fission.User.CID.Query
-import Fission.User.CID.Mutation
+import Fission.User.CID.Retriever
+import Fission.User.CID.Creator
+import Fission.User.CID.Destroyer
```

Comment:
Saves us from needing so many explicit imports when doing a lot of stuff with these classes.

```haskell
import qualified Fission.User.CID as User.CID

-- ...

foo :: (User.CID.Retriever m, User.CID.Destroyer m) => m Foo
```